### DISCUSSION TITLE
3 idées 'facture avoir,

### DISCUSSION THREAD

**Discussion by pwepwe973** [baimard/gestion#311] [Ideas]
déjà je dis merci pour ce merveilleux bijoux qui émerge qui était très attendu par moi même :) et qui va vous emmener très loin dans quelque années je le souhaite

je propose ces améliorations :

1- en cas de remboursement il serait bien de pouvoir éditer une facture d'avoir!

2- ajouter le prix d'achat de produit si il y'a un coût d'achat afin de pouvoir calculer les bénéfice à laide de ce petit bijoux

3 - une liste déroulante avec les mode de paiement que l'on propose.

j'ai d'autre idée mais ce serait trop long lol

courage pour le développement de ce petit bijoux!

cordialement SYLVAIN

**Comment by baimard** [comment]
Bonjour,
Merci de ton enthousiasme, je travail fort à l'amélioration. Mais pour ne pas tuer le bébé dans l'oeuf il va falloir que je me penche sur les problématiques de facture électronique :(

**Reply by les-les** [reply]
ad 1) 

créditer la facture et émettre une nouvelle facture

create a credit note for "false invoice" and send a new invoice 
have a look how the law especially VAT handles this issue



ad2) 

si vous mettez ces fonctionnalités dans ce projet, ce projet sera hors de proportion / et hors de l'eau
parce que vous devez toucher à l'outil et surtout bien comprendre l'ensemble de ces 3 parties
alors laissez ça tranquille pour l'instant...

if you put these feature in this will blow this project out of proportion / and out of the water
because you have to touch implement and most important fully understand all of ths 3 parts 
so leave this alone for now ... 


merchandise management / gestion des marchandises

calculation / calcul
 
stock valuation /  évaluation des stocks


ad3) 

a dropdown feature would be super handy in manx ways 

for payment methodes in invoice
for how long the bid/offert/devid is valid 
for discount and the terms of discount 

une fonction de liste déroulante serait super pratique de manière mannoise

pour les modes de paiement sur facture
pendant combien de temps l'enchère/offre/devid est valide
pour la remise et les conditions de remise

**Reply by les-les** [reply]
> Bonjour, Merci de ton enthousiasme, je travail fort à l'amélioration. Mais pour ne pas tuer le bébé dans l'oeuf il va falloir que je me penche sur les problématiques de facture électronique :(

invoicing is much more cmplex concerning the legal requirements than offert/bid/devis
both of them require a certain base data set
if the bid/offert/devis part is working flawless there will be an easy way to get invoicing working 


la facturation est beaucoup plus complexe concernant les exigences légales que l'offre/l'offre/le devis
les deux nécessitent un certain ensemble de données de base
si la partie offre/offre/devis fonctionne parfaitement, il y aura un moyen facile de faire fonctionner la facturation
